### PR TITLE
Add shardTotal catching

### PR DESCRIPTION
**Pull request by thehydrogen**
Makes sure that shardTotal is a number or "auto"

**Please describe the changes this PR makes and why it should be merged:**
Prevents possible errors from occuring.

**What does this PR do:**
- [ ] This PR changes internal functions, modules and/or utilities
  - [ ] This PR modifies the Extension API
- [ ] This PR modifies commands
  - [ ] This PR changes command functions
  - [ ] This PR changes metadata for the command, updated in `commands.js`
  - [ ] This PR removes and/or adds commands
- [ ] This PR modifies Web processing and/or content
  - [ ] This PR modifies static/ejs content
  - [ ] This PR modifies request processing (endpoint functions)
  - [ ] This PR modifies paths to existing resources
- [ ] This PR **only** includes non-code changes, like changes to default configurations, README, typos, etc.

### CODE REVIEW

File: master.js [hunk 1]
```diff
@@ -96,6 +96,10 @@ database.initialize(configJS.databaseURL).catch(err => {
 		}
 
 		winston.silly("Confirming shardTotal config value.");
+		if(Number.isInteger(configJS.shardTotal) === false && configJS.shardTotal !== "auto") {
```
Comment (vladfrangu) [inline master.js]: Could be `if (!parseInt(configJS.shardTotal) || configJS.shardTotal !== "auto")`

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: master.js (hunk 1 above)
Comment (wasdennnoch) [reply master.js]: *cough* `if (!parseInt(configJS.shardTotal) && configJS.shardTotal !== "auto")`

Comment (vladfrangu) [reply master.js]: Why? if it isn't a number or it isn't auto. I mean i don't think it matters that much

Comment (wasdennnoch) [reply master.js]: With ||, if it's not a number it errors, even if you set it to auto.

Comment (vladfrangu) [reply master.js]: Oh right JS.

Comment (thehydrogen) [reply master.js]: I'll change it.

**Comment by thehydrogen** [comment]
@KingDGrizzle I've applied the requested changes.

**Comment by vladfrangu** [comment]
That's not what we requested! We told you to change your added if check, not the one below 🤦‍♂️

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**